padinfo/padinfo.py

```diff
@@ -174,6 +174,23 @@ def get_monster_by_no(self, monster_no: int):
             await self._do_idmenu(ctx, m, self.id_emoji)
         else:
             await self.bot.say(self.makeFailureMsg(err))
+    
+    @commands.command(name="id2", pass_context=True)
+    async def _do_id_all_2(self, ctx, *, query: str):
+        """Monster info (main tab)"""
+        await self._do_id2(ctx, query)
+
+    @commands.command(name="idna2", pass_context=True)
+    async def _do_id_na(self, ctx, *, query: str):

```

Comment:
The naming conventions you're choosing here are very confusing.

Can you stick to:
old:
  _do_id_all()
  _do_id_na()
new:
 _do_id2_all()
 _do_id2_na()
or new:
 _do_id_all2()
 _do_id_na2()

Personally I prefer the first 'new' one even though it doesn't match the idna2 pattern. Maybe consider renaming it to id2na, but no strong preference. 

I'm not sure how useful these things are nowadays tbh, the na/jp delta is much shorter than it used to be.

Reply:
done & i changed to id2na